apps/web/src/server/views/applications/case/overview.njk

```diff
@@ -0,0 +1,56 @@
+{% extends "./applications-case-layout.njk" %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+
+{% block sectionContent %}
+	{% set applicantFirstOrganisationName = application.applicants[0].organisationName %}
+	{% set applicantFirstEmail = application.applicants[0].email %}
+	{% set applicantFirstPhoneNumber = application.applicants[0].phoneNumber %}
+	{% set applicantFirstWebsite = application.applicants[0].website %}
+	{% set userIsInspector = domainType === 'inspector' %}
+
+	<h3 class="govuk-heading-m">Latest key dates</h3>
+	<ul class="govuk-list govuk-list--bullet">
+		<li>Anticipated submission date: {{ application.keyDates.submissionDatePublished }}</li>
+		<li>Anticipated submission date internal: {{ application.keyDates.submissionDateInternal | datestamp({format: 'dd/MM/yyyy'}) }}</li>
+	</ul>
+
+	{% if not userIsInspector %}
+		<a class="govuk-link" href="#">Update key dates</a>
+	{% endif %}
+
+	<hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+	<h3 class="govuk-heading-m">Summary information</h3>
+	{{ govukSummaryList({
+		classes: 'govuk-summary-list--no-border',
+		rows: [
+			{
+				key: { text: "Case reference" },
+				value: { text: application.reference }
+			},
+			{
+				key: { text: "Applicant Information" },
+				value: { html: [applicantFirstOrganisationName, applicantFirstEmail, applicantFirstPhoneNumber] | collapse() }
+			},
+			{
+				key: { text: "Applicant website" },
+				value: { html: '<a class="govuk-link" href="//' + applicantFirstWebsite + '">' + applicantFirstWebsite + '</a>' }
```

Comment:
when applicant website is not defined the nunjucks render the word 'undefined' or 'null'
![immagine](https://user-images.githubusercontent.com/101253567/193060616-83888d20-e801-4ba7-9175-74f1a0b5dd07.png)